### PR TITLE
Releases version 2020.09.1

### PR DESCRIPTION
**Pull request by justinorrplus3it**
Fixes # .

Changes offered/proposed in this pull request:
- Version change from 2020.08 to 2020.09
- 
- 

* New PR Alert to: @plus3it/spel

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
 ## Changelog
+### 2020.09.22
+
+**Released**: 2020.09.22
+
+**Commit Delta**: [Change from 2020.07.1 release](http://github.com/plus3it/spel/compare/2020.08.1...2020.09.1)
```
Comment (eemperor) [inline CHANGELOG.md:6]: Change from version should be `2020.08.1`

```suggestion
**Commit Delta**: [Change from 2020.08.1 release](http://github.com/plus3it/spel/compare/2020.08.1...2020.09.1)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2020.09.rc1.1/manifests>
```
Comment (eemperor) [inline CHANGELOG.md:8]: Remove `rc1`

```suggestion
**Manifests**: <https://github.com/plus3it/spel/blob/2020.09.1/manifests>
```

**Review by eemperor — CHANGES_REQUESTED** [review]
You need to add the manifest files and also run bumpversion to update the version

**Comment by eemperor** [comment]
The PR title and commit message should also have the version number.

`Releases version 2020.09.1`

**Comment by lorengordon** [comment]
@justinorrplus3it for comparison, here is the pr for the last release: https://github.com/plus3it/spel/pull/415/files

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,16 @@
 ## Changelog
+### 2020.09.1
```
Comment (eemperor) [inline CHANGELOG.md:2]: Minor nitpick.  Add line before version heading.

```suggestion

### 2020.09.1
```

**Review by eemperor — CHANGES_REQUESTED** [review]

**Review by lorengordon — CHANGES_REQUESTED** [review]
There's something going on with the submodule that is not supposed to be here. Please do not merge until we get that fixed. Go ahead and make any changes to the pr, then I'll fix it up.

**Comment by eemperor** [comment]
Weird.  Thought it was cleaned up.

**Review by lorengordon — APPROVED** [review]
Looks good now, thanks!

**Review by eemperor — APPROVED** [review]

**Pull request merged**